reference/6/Microsoft.PowerShell.Core/Disable-PSRemoting.md

```diff
@@ -20,220 +20,250 @@ Disable-PSRemoting [-Force] [-WhatIf] [-Confirm] [<CommonParameters>]
 ```
 
 ## DESCRIPTION
-The **Disable-PSRemoting** cmdlet prevents users on other computers from running commands on the local computer.
 
-**Disable-PSRemoting** blocks remote access to all session configurations on the local computer.
-This prevents remote users from creating temporary or persistent sessions to the local computer.
-**Disable-PSRemoting** does not prevent users of the local computer from creating sessions (**PSSessions**) on the local computer or remote computers.
-
-To re-enable remote access to all session configurations, use the Enable-PSRemoting cmdlet.
-To enable remote access to selected session configurations, use the *AccessMode* parameter of the Set-PSSessionConfiguration cmdlet.
-You can also use the Enable-PSSessionConfiguration and Disable-PSSessionConfiguration cmdlets to enable and disable session configurations for all users.
+The `Disable-PSRemoting` cmdlet prevents users on other computers from running PSCore commands on
+the local computer.
+
+`Disable-PSRemoting` blocks remote access to all PSCore session configurations on the local
+computer. This prevents remote users from creating temporary or persistent PSCore sessions to the
+local computer. `Disable-PSRemoting` does not prevent remote users from creating Windows PowerShell
+sessions to the local computer. To disable all PowerShell session configurations, including Windows
+PowerShell configurations, run `Disable-PSRemoting` in Windows PowerShell on the local computer.
+`Disable-PSRemoting` does not prevent users of the local computer from creating sessions
+(**PSSessions**) on the local computer or remote computers.
+
+To re-enable remote access to all PSCore session configurations, use the `Enable-PSRemoting` cmdlet.
+To re-enable remote access to all session configurations, including Windows PowerShell
```

Comment:
Why are we duplicating this information for "Windows PowerShell" and "PowerShell v6"? This article is only about v6+ and the commands are the same regardless of version.

Reply:
This is a security issue.  This cmdlet on PSCore6 is currently broken because it affects only PSCore6 remote endpoints, but the cmdlet was intended to affect all 'PowerShell' remote endpoints.  A user may think he/she safeguarded their machine by running this cmdlet, when in fact only PSCore6 endpoints are affected and all Windows PowerShell endpoints remain enabled.

Since Windows PowerShell continues to be installed on all Windows machines, I feel the documentation should keep information about how this cmdlet behaves between the different versions.